apps/docs/src/components/LeftSidebar/LeftSidebar.astro

```diff
@@ -25,7 +26,50 @@ const sidebar = SIDEBAR[langCode];
             <h2 class="mb-2 font-bold leading-none text-lg">{header}</h2>
             <ul class="list-none p-0">
               {children.map((child) => {
+                const shouldBeExpanded = child.sublinks?.some(
+                  ({ link }) => link === currentPageMatch
+                );
+
+                if (child.sublinks) {
+                  return (
+                    <div>
+                      <h2 class="flex leading-none text-md ml-2 font-black mb-1 cursor-pointer w-full subTitle">
```

Comment:
i think the font is too big, lets stick to the normal size with chevron next to it

Reply:
also that should be `<li />` element to be sematically correct

Reply:
Right, done